src/rules/noNonNullAssertionRule.ts

```diff
@@ -24,7 +24,35 @@ export class Rule extends Lint.Rules.AbstractRule {
     public static metadata: Lint.IRuleMetadata = {
         ruleName: "no-non-null-assertion",
         description: "Disallows non-null assertions using the `!` postfix operator.",
-        rationale: "Using non-null assertion cancels the benefits of the strict null checking mode.",
+        rationale: `
```

Comment:
Did you forget `Lint.Utils.dedent` here?

Reply:
Haha yes, I did.